### PR TITLE
Add FSDN links.

### PR DESCRIPTION
**Pull request by kekyo**
Hi,
I discussion about require add FDSN's link for fsfoundation site (in seattle F# users group mvpsummit meetup).
I leave seattle and got agreement of linking fsfoundation and FSDN for site admins (@bleis-tift and @hafuu).

Thanks.

### CODE REVIEW

File: about/index.md [hunk 1]
```diff
@@ -64,3 +64,6 @@ the standard F# library.
 
  * [Data Access with F#](/guides/data-access)
 
+## FSDN - You can search F# API at yourself
```
Comment (dsyme) [inline about/index.md]: I think this should go on the "Learning" page, could you move it there?

Also change ``You can search F# API at yourself`` to just ``Search F# APIs`` please

Comment (kekyo) [inline about/index.md]: @dsyme  To learn.md?

Comment (ReedCopsey) [inline about/index.md]: That makes sense to me - I'd list it under the cheat sheet, since it somewhat fits along with those resources.

**Comment by kekyo** [comment]
@dsyme @ReedCopsey Is this OK?

**Comment by dsyme** [comment]
looks ok to me

**Pull request merged**